### DISCUSSION TITLE
about adding microsoft account

### DISCUSSION THREAD

**Discussion by cosettefantine** [ente-io/ente#3250] [Q&A] [answered]
How can i generate QR code so that scan it with ente on my microsoft account, I don't see that option. Is it possible to use ente with microsoft account?

**Comment by Damariobros** [comment, marked as answer]
You want to use Ente to sign in on your Microsoft account?

In that case, go into your Security Info settings ("manage how I sign in") and click "Add a new way to sign in or verify." Click on the option that says "Use An App." A screen will popup for Microsoft Authenticator, and there should be a link that says something along the lines of "use another authenticator app instead." This will take you through the setup for a standard authenticator app, and will give you a QR code to scan to put into Ente Auth.

Note that you can have both Microsoft Authenticator and a third-party authenticator app active at the same time; in that case, both the 8 digit code generated in Microsoft Authenticator and the 6 digit code generated by the third party one, in this case Ente Auth, will work at the same time, and the Microsoft Authenticator sign-in notifications method will also work.

**Comment by pizzav-xyz** [comment]


**Reply by Damariobros** [reply]
You shouldn't post your QR code without blocking it out. The QR code contains both the secret key that is used to generate your 2fa codes, and the email used to sign into your Microsoft account.

**Comment by cosettefantine** [comment]
Hello I am the owner of this mail account. What is going on? Why am i
getting this emails? Are they trying to get into my ente/ mail account?
Please provide me an information. Why am I in this conversation.

**Reply by Damariobros** [reply]
This is the thread on GitHub where you posted your question about how to set up Ente to sign into your Microsoft account. You picked my answer as the best answer and the issue was closed.

Because you are the creator of this thread, GitHub subscribed you to receive emails for replies by default.

Another user has found this thread and, for some reason, replied to this thread with nothing more than a screenshot of their qr code in their Microsoft account 2fa setup, and I replied to them warning them that they shouldn't do that. You received emails for each of these replies because of the default email subscription to threads you create or participate in on GitHub.

You may unsubscribe by navigating back to your question on GitHub and clicking the "Unsubscribe" button. Hope this clears up any confusion!
